src/components/Card/Card.css

```diff
@@ -74,7 +79,7 @@
 }
 
 .card .avatar img.broken {
-  display: none !important;
+  visibility: hidden;
```

Comment:
Why it needed?

Reply:
This happens when I changed from a div to a button
![Coding](https://user-images.githubusercontent.com/29065069/63586331-ffdec080-c598-11e9-8cba-1ee0018122aa.PNG)
 `display:none` removes the `image` tag from the `DOM` so the user icon is displaced down. I don't know why this happens for buttons but that is why I used `visibility:hidden` so the `image` tag is hidden but not removed from the `DOM`